pkg/udev/common.go

```diff
@@ -39,6 +39,8 @@ const (
 	UDEV_PATH           = "DEVPATH"         // udev attribute to get device path
 	UDEV_WWN            = "ID_WWN"          // udev attribute to get device WWN number
 	UDEV_SERIAL         = "ID_SERIAL_SHORT" // udev attribute to get device serial number
+	UDEV_SERIAL_FULL    = "ID_SERIAL"       // udev attribute to get - separated vendor,model,serial
```

Comment:
Why is this variable called UDEV_SERIAL_FULL and not ID_SERIAL?

Reply:
ID_SERIAL_SHORT key is assigned to UDEV_SERIAL and ID_SERIAL is assigned to UDEV_SERIAL_FULL
using UDEV_SERIAL we get only serial no of a disk and it is frequently used. Using UDEV_SERIAL_FULL we get `-` separated vendor, model, serial and it is used once to check devlinks. Main reason to make it UDEV_SERIAL_FULL is UDEV_SERIAL returns actual serial of a disk  and frequently used.